src/style.css

```diff
@@ -0,0 +1,91 @@
+/* Fonts */
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir.ttc);
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: bold;
+}
+
+html {
+  font-family: "Avenir", sans-serif;
+  font-size: 14px;
+  font-weight: 300;
+}
+
+.search-bar {
+  display: flex;
+  align-items: center;
+  position: relative;
+  margin-top: 20px;
+}
+
+.search-bar-button {
+  padding: 0;
+  display: block;
+  position: absolute;
+}
+
+.search-bar-button--big {
+  width: 19px;
+  height: 19px;
+  left: 26px;
+}
+
+.search-bar-button--small {
+  width: 11px;
+  height: 11px;
+  left: 13px;
+  top: 15px;
+}
+
+.search-bar-box {
+  display: block;
+  outline: none;
+  font-family: "Avenir", sans-serif;
```

Comment:
You've already set the font-family in html tag

Reply:
if I remove the font-family from this class, it won't apply